Lanchonete/LanchoneteTests/Managers/IngredientManagerTests.swift

```diff
@@ -21,5 +21,48 @@ class IngredientManagerTests: XCTestCase {
         
         XCTAssertTrue(ingredientManager.containsIngredient(ingredient: .bacon))
     }
+    
+    func test_GIVEN_a_list_with_6_bacons_WHEN_I_want_to_get_the_quantity_THEN_I_should_get_6 (){
+        let ingredientManager = IngredientManager(with:[.bacon,.bacon])
+        
+        XCTAssertEqual(ingredientManager.getQuantity(for: .bacon), 2)
+    }
+    
+    func test_GIVEN_a_list_with_a_bacon_and_a_egg_WHEN_I_want_to_get_the_quantity_for_the_egg_THEN_I_should_get_1 (){
+        let ingredientManager = IngredientManager(with:[.bacon,.egg])
+        
+        XCTAssertEqual(ingredientManager.getQuantity(for: .egg), 1)
+    }
+    
+    func test_GIVEN_an_empty_list_WHEN_I_want_to_get_the_quantity_for_bacon_THEN_I_should_get_0() {
+        let ingredientManager = IngredientManager(with:[])
+        
+        XCTAssertEqual(ingredientManager.getQuantity(for: .bacon), 0)
+    }
+    
+    func test_GIVEN_a_list_that_has_bacon_bacon_egg_WHEN_I_add_another_egg_THEN_the_list_should_be_bacon_bacon_egg_egg(){
+        let ingredientManager = IngredientManager(with:[.bacon,.bacon,.egg])
+        ingredientManager.addIngredient(.egg)
+        XCTAssertEqual(ingredientManager.getAllIngredients(), [.bacon,.bacon,.egg,.egg])
+    }
+    
+    func test_GIVEN_a_list_that_has_bacon_bacon_egg_WHEN_I_add_a_hamburger_THEN_the_list_should_be_bacon_bacon_egg_hamburger(){
+        let ingredientManager = IngredientManager(with:[.bacon,.bacon,.egg])
+        ingredientManager.addIngredient(.hamburger)
+        XCTAssertEqual(ingredientManager.getAllIngredients(), [.bacon,.bacon,.egg,.hamburger])
+    }
+    
+    func test_GIVEN_an_empty_list_WHEN_I_add_hamburger_THEN_the_list_should_be_hamburger() {
+        let ingredientManager = IngredientManager(with:[])
+        ingredientManager.addIngredient(.hamburger)
+        XCTAssertEqual(ingredientManager.getAllIngredients(), [.hamburger])
+    }
+    
+    func test_GIVEN_a_list_that_has_bacon_bacon_egg_WHEN_I_remove_one_hamburger_THEN_the_list_should_be_bacon_bacon_egg() {
+        let ingredientManager = IngredientManager(with:[.bacon,.bacon, .egg])
+        ingredientManager.removeIngredient(.hamburger)
+        XCTAssertEqual(ingredientManager.getAllIngredients(), [.bacon,.bacon, .egg])
+    }
```

Comment:
This tests removing an ingredient that does not exist. Where's the test to remove an ingredient that does exist? 😀

Reply:
totally forgot the happy path on this one 😄  ill add it now